common/web/keyboard-processor/build.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+#
+# Compile KeymanWeb's 'keyboard-processor' module, one of the components of Web's 'core' module.
+#
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+action=
+function action_failure() {
+  if [ -n "$action" ]; then
+    builder_report failure $action
+  fi
+}
+
+trap action_failure err
```

Comment:
I tinkered around a bit and figured out how to allow nicely-readable action-failure commands without squashing script errors as a side-effect!

This does require an extra local script variable or two, but nothing too egregious.

Example output:

![image](https://user-images.githubusercontent.com/25213402/190953160-594ed874-c26e-4ace-b01a-1e9b947a36e3.png)